APE2.rst

```diff
@@ -149,22 +187,8 @@ N/A
 Implementation
 --------------
 
-If this APE is accepted, a few organizational steps will be required:
-
-* The currently under-development version "v0.4" will be the first following
-  the release cycle.  Because v0.3 was released On the last Monday in Nov 2013,
-  the feature freeze and beta for v0.4 will be scheduled for the last Monday in
-  April 2014, With a release at the end of May 2014.  The milestone on GitHub
-  will thus need to be updated to match this.
-* The first LTS release (v1.0) will then come six months after v0.4, so an
-  appropriate milestone will need to be added to GitHub.  Following that,
-  milestones should be added as needed to reflect the versioning scheme
-  described in this APE.
-* The documentation should be updated to mention the existence of LTS releases
-  and make it clear that they come with a promise of two years of bug fixes (as
-  needed). It should also add a link to this APE so that those interested in the
-  numbering scheme can look here for details.
-
+Whenever this APE is updated, the core package documentation should be updated
```

Comment:
I would be nice to see a draft PR now over at `astropy` that shows me how the release procedures would change to comply with these changes here. Like... Will we be down to one backport branch? Do we no longer need a `stable` branch? Do we no longer need a `lts` build on RTD?

Reply:
Sounds good, I can try and do this

Reply:
Done